Packs/CortexAttackSurfaceManagement/Scripts/InferWhetherServiceIsDev/InferWhetherServiceIsDev.yml

```diff
@@ -1,15 +1,19 @@
 args:
-- description: 'Array of key-value objects. Each object within the array must contain the keys "Key" and "Value" to be considered. The values associated with those keys can be arbitrary. Example: [{"Key": "env", "Value": "dev"}, {"Key": "Name", "Value": "ssh-ec2-machine-name"}]'
+- description: 'Array of key-value objects. Each object within the array must contain the keys "Key" and "Value" to be considered. The values associated with those keys can be arbitrary. Example: [{"Key": "env", "Value": "dev"}, {"Key": "Name", "Value": "ssh-ec2-machine-name"}].'
   isArray: true
   name: asm_tags
-- description: 'Array of strings representing the Xpanse ASM "active classifications" for the service. Example: ["RdpServer", "SelfSignedCertificate"]'
+- description: 'Array of strings representing the Xpanse ASM "active classifications" for the service. Example: ["RdpServer", "SelfSignedCertificate"].'
   isArray: true
   name: active_classifications
+- description: infrastructure hierarchy information to include CSPs.
```

Comment:
i'm not sure i understand what this means, can you elaborate slightly?

Reply:
[PR feedback](https://github.com/demisto/content/pull/30601/commits/cc59bddcbcfddffbc2e558a72a565654b0bfa5d1)